packages/examples/examples/webpack/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "webpack-snap",
+  "version": "0.0.0",
+  "private": true,
+  "description": "An example Snap built using TypeScript and Webpack",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/snaps-skunkworks.git"
+  },
+  "license": "ISC",
+  "main": "src/index.js",
+  "files": [
+    "dist/",
+    "snap.manifest.json"
+  ],
+  "scripts": {
+    "build": "webpack",
+    "manifest": "mm-snap manifest --fix",
+    "eval": "mm-snap eval -b dist/snap.js",
+    "serve": "concurrently \"webpack --watch\" \"http-server -c-1 .\""
+  },
+  "devDependencies": {
+    "@babel/core": "^7.17.10",
+    "@babel/preset-typescript": "^7.16.7",
+    "@metamask/snap-types": "^0.14.0",
+    "@metamask/snaps-cli": "^0.14.0",
+    "@metamask/snaps-webpack-plugin": "^0.14.0",
+    "@types/node": "^17.0.33",
+    "@types/webpack-dev-server": "^4.7.2",
+    "assemblyscript": "^0.19.23",
```

Comment:
Why does this package have all these dependencies which I assume are unused?

Reply:
Copied the package.json from another example and forgot to remove those.